src/stc/ScintillaWX.cpp

```diff
@@ -771,35 +774,58 @@ sptr_t ScintillaWX::WndProc(unsigned int iMessage, uptr_t wParam, sptr_t lParam)
             break;
 #endif
 
-      case SCI_GETDIRECTFUNCTION:
+        case SCI_GETDIRECTFUNCTION:
```

Comment:
Are these whitespace changes intentional?

Reply:
Yes, I fixed the indentation in this commit a968b37